README.md

```diff
@@ -1,5 +1,8 @@
 # Generic Pipelines
 
+> [!WARNING]  
+> This repository is in the process of being split up into standalone repositories.
```

Comment:
I think we can update this warning as the split up is finished(i think).

Reply:
I still want to look into workflows into actions as well, if possible.